README.md

```diff
@@ -1,38 +1,315 @@
 # Sparrow
+**Summary**
+Sparrow is an Elixir library for [push notifications](en.wikipedia.org/wiki/Push_technology#Push_notification).
+[![Build Status](https://travis-ci.org/esl/sparrow.svg?branch=master)](https://travis-ci.org/esl/sparrow)
+[![Coverage Status](https://coveralls.io/repos/github/esl/sparrow/badge.svg)](https://coveralls.io/github/esl/sparrow)
 
-**TODO: Add description**
+Currently it provides support for the following APIs:
+* [FCM v1](https://firebase.google.com/docs/cloud-messaging/)
+* [APNS](https://developer.apple.com/library/archive/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/APNSOverview.html#//apple_ref/doc/uid/TP40008194-CH8-SW1)
 
-**TODO: Redirect Erlang lager to Elixir logger**
+**Requirements**
+* Elixir 1.6 or higher
+* Erlang OTP 20.3.6 or OTP 21
 
-**TODO: Add publishing package to hex** 
+# Build sparrow config
+This section describes how to write a config file for Sparrow.
+If you wish to use just one of the following services, do not include the other in the config.
 
-**TODO: Check API diffs between these articles:** 
+```elixir
+config :sparrow, config: [fcm_config, apns_config]
+```
+**Config opts**
 
-[link](https://developer.apple.com/documentation/usernotifications/setting_up_a_remote_notification_server/generating_a_remote_notification)
-and
-[link](https://developer.apple.com/library/archive/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/PayloadKeyReference.html#//apple_ref/doc/uid/TP40008194-CH17-SW1)
+***FCM specific***
+- `:path_to_json` - is a path to a json file provided by FCM
 
-## Installation
+***APNS specific***
+- `:auth_type` - defines the authentication type, the allowed values are: `:token_based`, `:certificate_based`
 
-[![Build Status](https://travis-ci.com/esl/sparrow.svg?branch=master)](https://travis-ci.com/esl/sparrow)
-[![Coverage Status](https://coveralls.io/repos/github/esl/sparrow/badge.svg)](https://coveralls.io/github/esl/sparrow)
+    1 `:token_based` requres to set:
```

Comment:
Why an ordered list? And if you insist on having the list ordered, use `1.`, `2.` instead of `1` and `2` - this way it'll be properly formatted by markdown.

Reply:
`requres to set` -> `requires setting`

Reply:
fixed